.github/scripts/prepare_release.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Fail on first error.
+set -e
+
+releaseLevel="$1"
+
+oldVersion="$(node -pe 'require("./package.json").version')"
+npx standard-version --release-as "$releaseLevel" --skip.commit=true --skip.changelog=true --skip.tag=true
+
+cd selenium
+npx standard-version --release-as "$releaseLevel" --skip.commit=true --skip.changelog=true --skip.tag=true
```

Comment:
Why not use `.versionrc.json` so `standard-version` updates both the root and the selenium dir at the same time?

```json
{
  "commitAll": true,
  "bumpFiles": ["package.json", "selenium/package.json"],
  "skip": {
    "commit": true,
    "tag": true,
    "changelog": true
  }
}
```